### ISSUE TITLE
Add ability to set custom hour tracks?

### ISSUE THREAD

**Issue by AlecHCodes** [mn6/nook-desktop#36]
Hello! Longtime enjoyer of the browser plugin here, just found out it got deprecated in favour of a full blown app (still waiting on that Linux Build though, haha!).

I have a feature request: is it within the scope of the project (or even possible for that matter) to add a setting to select manual playback of our own hourly music loops?

This also seems kinda related to #5, I think.

**Comment by axelludwig** [comment]
yes and a way to choose sons from any time of the day randomly, and maybe a night time or day time mode as well
